sdk/devcenter/azure-developer-devcenter/src/samples/java/com/azure/developer/devcenter/generated/DevBoxesCreateDevBox.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.developer.devcenter.generated;
+
+import com.azure.core.http.rest.RequestOptions;
+import com.azure.core.util.BinaryData;
+import com.azure.core.util.polling.SyncPoller;
+import com.azure.developer.devcenter.DevBoxesClient;
+import com.azure.developer.devcenter.DevBoxesClientBuilder;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+
+public class DevBoxesCreateDevBox {
+    public static void main(String[] args) {
+        DevBoxesClient devBoxesClient =
+                new DevBoxesClientBuilder()
+                        .credential(new DefaultAzureCredentialBuilder().build())
+                        .devCenter("ContosoDevCenter")
+                        .devCenterDnsSuffix("devcenter.azure.com")
+                        .tenantId("84fq37b9-22t3-t887-l09p-93241ngq0s95")
+                        .buildClient();
+        // BEGIN:com.azure.developer.devcenter.generated.devboxescreatedevbox.devboxescreatedevbox
+        BinaryData body = BinaryData.fromString("{\"poolName\":\"LargeDevWorkStationPool\"}");
+        RequestOptions requestOptions = new RequestOptions();
+        SyncPoller<BinaryData, BinaryData> response =
+                devBoxesClient.beginCreateDevBox("myProject", "me", "MyDevBox", body, requestOptions);
+        // END:com.azure.developer.devcenter.generated.devboxescreatedevbox.devboxescreatedevbox
```

Comment:
If you've modified any of these samples, move it one level above (out of "generated"). Then you can modify the tag and include them in README.md

Reply:
These ones are unmodified (modified only in the ReadmeSamples.java)